src/modules/esl-utils/dom/events/listener.ts

```diff
@@ -16,8 +17,10 @@ export interface ESLListenerEventMap extends HTMLElementEventMap {
 export type ESLListenerDescriptor<EType extends keyof ESLListenerEventMap = string> = {
   /** Event type (name) */
   event: EType;
-  /** Use capture DOM Event phase */
+  /** Use capture DOM Event phase, see {@link AddEventListenerOptions.capture} for details */
```

Comment:
cosmetic: can we just use nouns here and below?
```suggestion
  /** Capture DOM Event phase, see {@link AddEventListenerOptions.capture} for details */
```

or mb `using` instead of `use`

Reply:
Updated using even better version from MDN :) Thx!